runatlantis.io/docs/automerging.md

```diff
@@ -29,6 +29,20 @@ Automerging can be enabled either by:
 If automerge is enabled, you can disable it for a single `atlantis apply`
 command with the `--auto-merge-disabled` option.
 
+## How to set merge method for automerge
+
+If automerge is enabled, you can use `--merge-method` option
+for `atlantis apply` command to specify which merge method use.
```

Comment:
```suggestion
## How to set the merge method for automerge

If automerge is enabled, you can use the `--merge-method` option
for the `atlantis apply` command to specify which merge method use.
```

Reply:
fixed in https://github.com/runatlantis/atlantis/pull/4895/commits/cf878039e12effa3f377c3d4076ee519ea2184ff